backend/src/plugins/ModActions/functions/banUserId.ts

```diff
@@ -97,6 +97,17 @@ export async function banUserId(
     };
   }
 
+  if (pluginData.config.get().delete_invites_on_ban) {
+    const userInvites = (await pluginData.guild.getInvites()).filter(inv => inv.inviter?.id === user.id);
+    userInvites.forEach(invite => {
```

Comment:
After adding `await` below for `invite.delete()`, it might be best to do a `for (const invite of userInvites)` loop instead and bail from the loop at the first deletion error. The current code would attempt to delete the invites in parallel due to `forEach` which could potentially cause a lot of errors.